src/Data/Decoder.elm

```diff
@@ -0,0 +1,52 @@
+module Data.Decoder exposing (..)
+
+import Data.Audience            exposing (Audience, AudienceType(..))
```

Comment:
mhhh where is elm-format? 😆

Reply:
Imagine you have to change it for all imports every time you add new long import not fitting to current space :)